roles/openshift_gcp/templates/openshift-bootstrap-update.j2

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -euo pipefail
+
+oc serviceaccounts create-kubeconfig -n openshift-infra node-bootstrapper > /root/bootstrap.kubeconfig
```

Comment:
Where does this run?  Should this `/root/bootstrap.kubeconfig` be in `/etc/origin/master`?

Reply:
This is the job that keeps the bootstrap metadata in project instances up to date.  So it's creating a new bootstrap config, uploading it to gcp metadata, then deleting the temporary local file.